src/main/java/org/opentripplanner/graph_builder/module/osm/OpenStreetMapModule.java

```diff
@@ -562,11 +562,30 @@ private VehicleParking createVehicleParkingObjectFromOsmEntity(
                     String.format("%s/%d", entity.getClass().getSimpleName(), entity.getId())
             );
 
+            var tags = new ArrayList<String>();
+
+            tags.add(isCarParkAndRide ? "osm:amenity=parking" : "osm:amenity=bicycle_parking");
+
+            if (entity.isTagTrue("fee")) {
+                tags.add("osm:fee");
```

Comment:
These "feed id scoped" tags make it easier to find what is the origin of the tag but it also makes it messy if we would want to reuse the same tags from a different source.

Reply:
We discussed this today in the meeting. This tag format is fine but you can add now or we might add later a "source:osm" tag so we could use that format if we only want to include osm parks in routing for example.

Reply:
There are two tags: `osm:amenity=parking`, `osm:amenity=bicycle_parking` which may be used as a `source:osm` tag.

Reply:
That's true but I wonder if it would be nice to have a standard format tag that could be reused for future updaters as well. It might be a bit cryptic for users to figure out what are the tags that are needed if only parkings from certain feed should be included. Although, I'm not sure if it would be more ideal to filter in/out parkings based on "feedId" through this requiredVehicleParkingTags or should there be a "allowedVehicleRentalNetworks" style query parameter for parkings as well.